src/psyclone/dynamo0p3_builtins.py

```diff
@@ -689,10 +717,10 @@ def gen_code(self, parent):
 
 
 class DynIncXTimesYKern(DynBuiltIn):
-    ''' Multiply the first field by the second and return it '''
+    ''' Multiply the first real-valued field by the second and return it '''
```

Comment:
Commas, full stop and blank line please.

Reply:
Yep.